hack/go-tools/verify-gofmt.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(cd "${SCRIPT_ROOT}/../.." && pwd)"
+pushd ${REPO_ROOT} > /dev/null
+
+find_files() {
+  find . -not \( \
+      \( \
+        -wholename '*/vendor/*' \
+        -o -wholename '*/*kubebuilder.go' \
+        -o -wholename '*/*deepcopy.go' \
+        -o -wholename '*/doc.go' \
```

Comment:
Why exclude `doc.go`?

Reply:
@irfanurrehman please refer to some discussion here https://github.com/kubernetes-sigs/federation-v2/pull/304#pullrequestreview-168197489